### PR TITLE
maintenance, tox, github actions

### PR DESCRIPTION
**Pull request by JakaKokosar**
##### Issue

https://github.com/biolab/orange3/issues/5145

### CODE REVIEW

**Comment by ajdapretnar** [comment]
List of issues to fix:
1. Timeout in Topic Modeling. I have not seen this issue yet. Could be already fixed in orange-widget-base master?
```
Traceback (most recent call last):
  File "/home/runner/work/orange3-text/orange3-text/.tox/py-orange-released/lib/python3.7/site-packages/orangecontrib/text/widgets/tests/test_owtopicmodeling.py", line 63, in test_all_topics_output
    self.process_events(until)
  File "/home/runner/work/orange3-text/orange3-text/.tox/py-orange-released/lib/python3.7/site-packages/orangewidget/tests/base.py", line 374, in process_events
    raise TimeoutError()
TimeoutError
```
2. AssertionError in Twitter test. Known issue. Works locally, sometimes fails on Travis. Re-run usually helps. Could be an issue of concurrency. 🤷 
```
Traceback (most recent call last):
  File "/home/runner/work/orange3-text/orange3-text/.tox/py-orange-released/lib/python3.7/site-packages/orangecontrib/text/widgets/tests/test_owtwitter.py", line 18, in test_empty_author_list
    self.assertTrue(self.widget.Error.empty_authors.is_shown())
AssertionError: False is not true
```
3. Could be an issue of failed NLTK data load? I've also not seen this issue before. `nltk.download()` has to be run before testing - I don't know how this is handled with tox.
`Error: Could not find or load main class edu.stanford.nlp.tagger.maxent.MaxentTagger`

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install --upgrade tox tox-pip-version
```
Comment (ajdapretnar) [inline .github/workflows/test.yml:79]: A wild guess, but I would say that after installing dependencies, `nltk.download()` should be run to download the data necessary for tests.

**Comment by PrimozGodec** [comment]
It should fix Lsi test errors https://github.com/biolab/orange3-text/pull/619

**Comment by PrimozGodec** [comment]
Twitter should be fixed here https://github.com/biolab/orange3-text/pull/616

**Pull request merged**